dist/s_evergreen_validate

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+t=__wt.$$
+trap 'rm -f $t' 0 1 2 3 13 15
+
+fast=1
+case "$1" in
+    -F) # Run fast.
+        fast=0
+        shift;;
+    *)
+        if test $# -gt 1 ; then
+            echo "Usage: $0 [-F]"
+            echo "-F only run validation commands if evergreen yml files have been modified"
+        fi
+        break;;
+esac
+
+if [ $(command -v evergreen) ] && [ -f ~/.evergreen.yml ]; then
+    if test $fast -eq 0 ; then
```

Comment:
Nit: I think code like this is clearer if you set `fast=true` or `fast=false`, and then this becomes:
```
  if $fast; then
```
(to get the negation you can do `if ! $fast; then`).